os/drivers/kernel_test/Make.defs

```diff
@@ -14,40 +14,16 @@
 # either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 #
-############################################################################
-# Include kernel test drivers
+###########################################################################
 
-ifeq ($(CONFIG_DRIVERS_KERNEL_TEST),y)
+-include $(TOPDIR)/Make.defs
+DELIM ?= $(strip /)
 
-CSRCS += kernel_test_drv.c
-CSRCS += test_task.c
-CSRCS += test_sched.c
-CSRCS += test_clock.c
-CSRCS += test_timer.c
-CSRCS += test_sem.c
-CSRCS += test_group.c
-
-ifneq ($(CONFIG_DISABLE_SIGNALS),y)
-CSRCS += test_signal.c
-endif
-
-ifeq ($(CONFIG_TC_COMPRESS_READ),y)
-CSRCS += test_compress_decompress.c
-endif
-
-ifeq ($(CONFIG_ARMV8M_TRUSTZONE),y)
-CSRCS += test_tz.c
-endif
+include kernel_test_codes$(DELIM)Make.defs
```

Comment:
You make this file is not included from drivers/Makefile. So, you should include only this file from drivers
In drivers/Makefile
```
include kernel_test$(DELIM)kernel_test_codes$(DELIM)Make.defs
include kernel_test$(DELIM)network_test_codes$(DELIM)Make.defs

=>

include kernel_test$(DELIM)Make.defs
```
And then keep this.

Reply:
corrected